pages/index.js

```diff
@@ -1,14 +1,12 @@
 import React from "react";
 import PedalGallery from "../components/pedal-gallery/index";
 import styled from "styled-components";
-import Link from "next/link";
 
-export default function HomePage() {
+export default function HomePage({ pedals, setPedals }) {
   return (
     <div>
       <MainHead>My Sound Journal</MainHead>
-      <PedalGallery />
-      <Link href="/new-pedal">Add a Pedal</Link>
+      <PedalGallery pedals={pedals} setPedals={setPedals} />
```

Comment:
we try to avoid drilling the setter Function. check teh state session again to handle the state. [here](https://github.com/neuefische/muc-web-23-3/blob/main/sessions/react-state-2/react-state-2.md#sharing-state-between-components)